spec/requests/api/visitor_can_see_specific_article_spec.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.describe 'GET/api/atricles' do
+  let!(:article) { create(:article) } 
+  describe 'successfully get a specific article' do
```

Comment:
What happens if I pass in 'abc' instead of a real id? 🤔  sad path is needed for this feature